src/pyfaf/storage/problem.py

```diff
@@ -133,44 +136,44 @@ def quality(self):
         return self.sorted_reports[0].quality
 
     @property
-    def sorted_reports(self):
+    def sorted_reports(self) -> List[Any]:
         '''
         List of all reports assigned to this problem
         sorted by quality.
         '''
         return sorted(self.reports, key=lambda r: r.quality, reverse=True)
 
     @property
-    def active_reports(self):
+    def active_reports(self) -> List[Dict[str, Any]]:
         '''
         List of all non archived reports
         '''
         return [r for r in self.reports if not r.archived]
 
     @property
-    def archived_reports(self):
+    def archived_reports(self) -> List[Any]:
         '''
         List of archived reports
         '''
         return [r for r in self.reports if r.archived]
 
     @property
-    def backtraces(self):
+    def backtraces(self) -> List[Report]:
         '''
         List of all backtraces assigned to this problem.
         '''
         return sum([x.backtraces for x in self.reports], [])
 
     @property
-    def sorted_backtraces(self):
+    def sorted_backtraces(self) -> List[Report]:
```

Comment:
Same thing here, you are sorting `Problem.backtraces`, which is `List[ReportBacktrace]`.